### DISCUSSION TITLE
Ease of use: `Command + K` to navigate the website

### DISCUSSION THREAD

**Discussion by dunstorm** [franken-ui/ui#23] [Ideas]
Can we add Command + K to navigate through the components just like ui.shadcn.com?

**Comment by dunstorm** [comment]
Potentially just a small change, we need to open the Search Box on `Command + K`

**Reply by sveltecult** [reply]
It is currently set to `Ctrl + /`

**Reply by dunstorm** [reply]
I see, thanks. Is it possible to set it to `Command + K` on Mac only just like shadcn? `Ctrl + /` is hard to reach in the keyboard.

**Reply by acemtp** [reply]
Would be cool to be able to customize the shortcut but it's hardcoded for now
